serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,274 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* https://adoptium.net/[JDK 11+] installed with `JAVA_HOME` configured appropriately
+* https://maven.apache.org/install.html[Apache Maven]
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI]
+
+To edit your workflows:
```

Comment:
It's better to link to the tooling guide: https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/getting-started/getting-familiar-with-our-tooling.adoc

Reply:
In this tooling tutorial, we should introduce Maven and Quarkus CLI since they are part of the developer toolkit.